### ISSUE TITLE
Train and evaluate NB and SVM

### ISSUE THREAD

**Issue by annamarianau** [keknight/DSE_511_project3#3]

**Comment by csmetzner** [comment]
Tasks:

- Perform hyperparameter tuning using GridSearchCV for Naive Bayes and Support Vector Machine Classifier for normalized data and dimensionality reduced data via PCA.
- Train both models on training data.
- Evaluate both models on unseen testing data.
- Investigate confusion-matrix (TP, FP, FN, TN)
- Analyze feature importance, i.e., get most important features. 
- Visualization (Plots) (TBD)
